test/specs/sdk.specs.ts

```diff
@@ -50,4 +50,25 @@ describe("SDK", () => {
                 .equal(sdk.api);
         }
     });
+
+    it("should add listener on the sdk", () => {
```

Comment:
Not really related, just added those tests as I wanted to know if some branches tests were missing